tools/c7n_mailer/c7n_mailer/azure/deploy.py

```diff
@@ -18,6 +18,8 @@
 import os
 import logging
 
```

Comment:
Looks like extra whitespace between imports that should be grouped (optimize imports)